### DISCUSSION TITLE
DSC Alexor PC9155

### DISCUSSION THREAD

**Discussion by lewisward1** [taligentx/dscKeybusInterface#316] [dscKeybusInterface Discussion] [answered]
This project looks amazing and exactly what i need. I noticed in the notes it say it will not work with PC9155 Alexor alarm as it has no output for that. I wondered if anyone knows a way which it could work if the alarm panel has the add on GS2020/GS2065 board and connected to the ethernet. Think this board is supposed to send info back to an alarm company but i installed mine myself years ago and never set that up (at the time i used the GSM to send text message to my phone)

Any help for the above appreceatied or if somebody can recommend another wireless model that is as reliable and secure as the Alexor has been but can connect to home assistant, please can you let me know the models. I am from UK so DSC support here is not really possible.

I dont really want to run any wires over the house - hence why i choose the wireless. I have seen companies like ring and yale do little boxes that can integrate with home automations but not sure how reliable and good they are. i was recommended DSC 

Thanks

**Comment by kricon** [comment, marked as answer]
You can replace just main PC9155 Alexor central unit with DSC PowerSeries panel (PC1616/PC1832/1864) with TR5164 wireless receiver so your wireless sensors and input devices are compatible, except wireless siren which will you need to wire up. So you can keep same keypads, smoke detectors, PIR's and magnet sensors etc just match their frequency (433/868Mhz) with your current devices..

TL265GS/GS2065 modules are intended for announcing, programming and communicating to Monitoring-Stations and they are connecting using PC-Link interface which is not supported by this project.

**Comment by lewisward1** [comment]
Nice one. Thanks so much for this !

**Comment by lewisward1** [comment]
would a RF5132 do the the same job ? it says this one is compatible with my siren (WT4911)

Or if not is their a siren i could buy that will speak to the panel you suggested without wires ?
